state/store_test.go

```diff
@@ -23,6 +24,16 @@ func getConfig() *config.Config {
 	}
 }
 
+func initConfig() {
+    os.Setenv("COOKIE_SECRET", "hello")
+    os.Setenv("COOKIE_NAME", "TEST")
+}
+
+func cleanupConfig() {
+    os.Unsetenv("COOKIE_SECRET")
+    os.Unsetenv("COOKIE_NAME")
```

Comment:
Why are the environment variables being modified? this should be controlled using mocking.

Reply:
Because we're using `config.Get()` in the code, we can't mock `config`, we don't pass it in to the `store`. To mock this, `Get()` would need to become a member of the `config` struct as opposed to a function inside the `config` package, `config` would need to be passed in to the `NewStore()` method, and that goes against what we've been directed.